### PR TITLE
Add instruction on how to have access to the bot in IssueGuidelines.md

### PR DESCRIPTION
**Pull request by damusnet**

### CODE REVIEW

**Comment by facebook-github-bot-0** [comment]
By analyzing the blame information on this pull request, we identified @mkonicek and @clickthisnick to be potential reviewers.

**Comment by christopherdro** [comment]
@facebook-github-bot shipit

**Comment by facebook-github-bot-2** [comment]
Thanks for importing. If you are an FB employee go to [Phabricator](https://our.intern.facebook.com/intern/opensource/github/pull_request/578376612321770/int_phab) to review.

File: docs/IssueGuidelines.md [hunk 1]
```diff
@@ -46,3 +46,6 @@ Comment `@facebook-github-bot reopen`
 
 ### What are all the available commands for the bot?
 When you mention the bot, it follows the commands defined in [IssueCommands.txt](https://github.com/facebook/react-native/blob/master/bots/IssueCommands.txt).
+
+### How can I help manage issues?
+Thank you for wanting to take the time to help us with managing GitHub issues. To have access to the bot please add your GitHub username to the first line of  [IssueCommands.txt](https://github.com/facebook/react-native/blob/master/bots/IssueCommands.txt), in alphabetical order, by submitting a Pull Request.
```
Comment (marketmar) [inline docs/IssueGuidelines.md]: Locked out of my MacBook Pro

Comment (marketmar) [inline docs/IssueGuidelines.md]: Help

**Pull request closed without merging**